javascript/javascript1/week3/task2.js

```diff
@@ -0,0 +1,17 @@
+// When will we be there??
+
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+  
+const travelTime = calcTravelTime(travelInformation);
+
+function calcTravelTime(travelInformation) {
+    const travelInMinutes = (travelInformation.destinationDistance / travelInformation.speed * 60);
+    const h = Math.floor(travelInMinutes / 60);
```

Comment:
naming is very important as it defines the value we are assigning to. don't use single letters

Reply:
Thank you for reviewing my homework. I changed them!